packages/shared/src/styles/constants.scss

```diff
@@ -51,6 +51,7 @@ $color-blue-1: #0dc2e7;
 $color-blue-2: #2a3052;
 $color-blue-3: #0796e0;
 $color-blue-4: #0677af;
+$color-blue-5: #377cfc29;
```

Comment:
I suggest reusing hintbox component but if you need to create a new one, follow the steps below.

1. You need to create an alpha color.
$alpha-color-blue-4: transparentize($color-blue, 0.16);

2. Perhaps you can reuse `--transparent-info` in `themes.scss` file.



I suggest you to read and understand the components code that you're working on before making changes.